interface/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+name: lxd
```

Comment:
A metadata.yaml doesn't exist for an interface in the operator framework world. It's just a python library, nothing else.